### PR TITLE
Automatic language detection

### PR DESCRIPTION
**Pull request by jackdavidweber**
Set up the frontend so that it can handle language detection once backend language detection is set up.

Also made sure that frontend sends a get request to backend to ping it so that it wakes up.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -84,20 +87,25 @@ export default function App() {
 
     async function postRequest(inputValue, inputLanguageValue, outputLanguageValue ){
         const params = new URLSearchParams({
-            input: inputValue,
-            in_lang: inputLanguageValue,
-            out_lang: outputLanguageValue
+          input: inputValue,
```
Comment (jackdavidweber) [inline src/App.js:87]: Note that i was just fixing a pre-existing indentation issue.

File: src/App.js [hunk 2]
```diff
@@ -84,20 +87,25 @@ export default function App() {
...
+          out_lang: outputLanguageValue
         })
 
         const requestOptions = {
-            method: 'POST',
-            headers: new Headers({
-                'Content-Type': 'application/x-www-form-urlencoded',
-            }),
-            body: params.toString()
+          method: 'POST',
```
Comment (jackdavidweber) [inline src/App.js:93]: pre-existing indentation issue

File: src/App.js [hunk 3]
```diff
@@ -65,10 +65,13 @@ export default function App() {
     async function getRequest() {
         const res = await fetch(`https://cjsback.herokuapp.com/`)
         const data = await res.json()
+        return data
```
Comment (jackdavidweber) [inline src/App.js]: felt that it was strange to not have get request return anything even though I am currently not doing anything with data

Comment (Danish21) [reply src/App.js]: Suggestion: As a general rule of thumb only expose only returns if it used or will be used immediately. Helps makes the code less confusing for posterity if the return is unused. Based on that I would remove the return statement.

Also a better name might be wakeUpServer for this function to highlight without having to comment that this is used exclusively to wake up the server.

File: src/App.js [hunk 4]
```diff
@@ -84,20 +87,25 @@ export default function App() {
...
+          headers: new Headers({
+              'Content-Type': 'application/x-www-form-urlencoded',
+          }),
+          body: params.toString()
         };
         const response = await fetch('https://cjsback.herokuapp.com/', requestOptions);    
         const translation = await response.json();
+        
+        // change in_lang only if it is defined
+        if (typeof translation['response_in_lang'] !== "undefined"){
```
Comment (Danish21) [inline src/App.js]: Why not 
`
translation['response_in_lang'] !== undefined
`
?

**Review by Danish21 — APPROVED** [review]

File: src/App.js (hunk 3 above)
Comment (cjoshea9) [reply src/App.js]: +1 on wakeUpServer

**Review by cjoshea9 — APPROVED** [review]

File: src/App.js (hunk 3 above)
Comment (jackdavidweber) [reply src/App.js]: got it!

File: src/App.js (hunk 4 above)
Comment (jackdavidweber) [reply src/App.js]: Just switched it - I did it using typeof because of a stack overflow answer but this works as well.

**Pull request merged**